programmers/20210504/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    var answer = '';
+    return s.length % 2 ? s[parseInt(s.length / 2)] : s[s.length / 2 - 1] + s[s.length / 2];
```

Comment:
괄호를 이용해주시면 더 가독성이 좋을거 같아요!
```suggestion
    return s.length % 2 ? s[parseInt(s.length / 2)] : s[(s.length / 2) - 1] + s[s.length / 2];
```